examples/test_extract.py

```diff
@@ -38,5 +36,7 @@ def main(args):
     parser.add_argument("-o", "--output-file", type=str, help="Output KRec file path (optional)")
     parser.add_argument("-v", "--verbose", action="store_true", help="Enable verbose output")
 
+    logging.basicConfig(level=logging.INFO, format="%(asctime)s - %(levelname)s - %(message)s")
```

Comment:
actually, you can add `colorlogging` as a dependency, it is a small package i wrote to make it easy to configure nice logging: https://pypi.org/project/colorlogging/